F/Flex/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Flex"
+version = v"2.6.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/westes/flex.git", "d69a58075169410324fe49666f6641ba6a9d1f91")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd flex
+apk add gettext-dev
+apk add texinfo
+apk add help2man
+./autogen.sh
+./configure --prefix=${prefix}
+make -j${nprocs}
+make install
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:glibc)
```

Comment:
Will change this to `Platform...` with the subsequent commit adding most of the platforms